cukinia.conf.sample

```diff
@@ -6,11 +6,13 @@ cukinia_process Xorg
 cukinia_process nginx www-data
 cukinia_python_pkg math
 cukinia_http_request www.google.com
+cukinia_mount "/dev/sda" "/boot "
 
 # Those tests should always fail
 cukinia_process nonexistent nonexistent
 cukinia_user nonexistent
 cukinia_http_request http://127.0.0.1:81/
 cukinia_python_pkg nonexistent
+cukinia_mount nonexistant nonexistant nonexistant
```

Comment:
* it's nonexist**e**nt (see above)
* please use things that highlight how the tests works, eg. 
 cukinia_mount /dev/nodev /somedir noopt